letsencrypt/storage.py

```diff
@@ -127,6 +122,59 @@ def get_link_target(link):
     return os.path.abspath(target)
 
 
+def _relevant(option):
+    """
+    Is this option one that could be restored for future renewal purposes?
+    :param str option: the name of the option
+
+    :rtype: bool
+    """
+    # The list() here produces a list of the plugin names as strings.
+    from letsencrypt import cli
```

Comment:
is it bad that these are imported in a helper function? Will that cause them to get imported multiple times per call to relevant_vaules or does python do some optimazation for this